app/code/community/Paylike/Paymentmethod/Model/Paymentmethod.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+require_once dirname(__FILE__) . '/../Model/Paylike.php';
+class Paylike_Paymentmethod_Model_Paymentmethod extends Mage_Payment_Model_Method_Cc
+{
+    const REQUEST_TYPE_AUTH_CAPTURE = 'AUTH_CAPTURE';
+    const REQUEST_TYPE_AUTH_ONLY = 'AUTH_ONLY';
+
+    protected $_isGateway = true;
+    protected $_canAuthorize = true;
+    protected $_canUseCheckout = true;
+    protected $_canCapture = true;
+    protected $_canCapturePartial = false;
+    protected $_canRefund = true;
+    protected $_canRefundInvoicePartial = true;
+    protected $_canVoid = true;
+    protected $_code = 'paymentmethod';
+    protected $_formBlockType = 'paymentmethod/form_paymentmethod';
+    protected $_infoBlockType = 'paymentmethod/info_paymentmethod';
+
+    public function authorize(Varien_Object $payment, $amount)
+    {
+        $payment->setTransactionId($payment->getPaylikeTransactionId());
+        $payment->setIsTransactionClosed(0);
+        return $this;
+
+    }
+
+    public function capture(Varien_Object $payment, $amount)
+    {
+        $arr = array('amount' => $amount * 100,
+            'currency' => Mage::app()->getStore()->getCurrentCurrencyCode(),
+            'descriptor' => 'Capture',
```

Comment:
This is the text that will appear on the customer's credit card statement. It should help the customer remember what the transaction is about. This will not help at all as all charges on the statement are captures by definition.
Remove it from here and we will then fall back to the descriptor on the merchant account.